packages/amplify-provider-awscloudformation/package.json

```diff
@@ -28,6 +28,7 @@
     "extract-api": "ts-node ../../scripts/extract-api.ts"
   },
   "dependencies": {
+    "@aws-amplify/amplify-category-auth": "3.0.3",
```

Comment:
It didn't really work because the typing in auth is for the user interfaces. I think I can lift these functions and types out of auth category and put them into the provider-cloudformation package with the added bonus that this functionality is now more modular and better tested 👍🏼

Reply:
sounds good.